06_correlation_analysis.ipynb

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    y_df.head()
</pre><p>In case it's helpful, sometime I use the function <code>display</code> (already loaded by Jupyter) to show several things within the same cell. Not sure if <code>print</code> nicely prints a data frame too.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Good idea! I do think <code>display</code> would be a good choice here. <code>print</code> doesn't seem to format the DFs as tables the same way, it just prints them as text.  </p>